source/fundamentals/crud/read-operations/retrieve.txt

```diff
@@ -0,0 +1,121 @@
+==============
+Retrieve Data
+==============
+
+.. default-domain:: mongodb
+
+
+Read operations allow you to retrieve data from your MongoDB database.
+
+You can retrieve data using find, aggregate, and watch operations.
+
+A find operation is useful if you want a subset of the existing data
+based on your query filter. You can further specify which documents to
+retrieve such as how to sort results, skip results, limit results and
+limit the fields to return. 
+
+An aggregate operation is useful if you want to perform an ordered
+pipeline of transformations to the matched data. You can perform all
+find operations, rename fields, calculate fields, summarize data and
+group values.
+
+A watch operation is useful if you want to monitor the database in
+real-time. You can specify which write operations to monitor such as
+insert, update and delete.
+
+The following examples use documents in the ``paint_orders`` collection
+which represent invoices of a paint store:
+
+.. code-block:: json
+
+    { "_id": 1, "color": "red", "qty": 16 }
+    { "_id": 2, "color": "purple", "qty": 10 }
+    { "_id": 3, "color": "green", "qty": 8 }
+    { "_id": 4, "color": "purple", "qty": 4 }
+    { "_id": 5, "color": "red", "qty": 9 }
+    { "_id": 6, "color": "green", "qty": 11 }
+
+Find
+----
+
+The find operation allows you to specify a query to show specific results
+using the ``find()`` method. This method makes a call to the collection
```

Comment:
I think this conflates "the find operation" with "the find method". Maybe this could be broken up into two sentences?

I think it is incorrect to say "call to the collection". I believe find triggers communication with the user's MongoDB instance. find() is a method of MongoCollection, but I do not believe it is accurate to say that the method makes a call to a collection.